src/ts/pages/DataCatalog/__tests__/DataCatalog.test.tsx

```diff
@@ -413,11 +415,12 @@ describe('DataCatalog', () => {
       })
 
       setupMockResponse('keyword=C002-FAKE', 1, 1, 'Found ')
+      const nextButton = await screen.findByRole('button', { name: 'Next' })
```

Comment:
I split these out because I thought the issue was with the test but, the fix was on the vitest.config I think I prefer it better this way anyways